src/Executor/CommandLine.cs

```diff
@@ -0,0 +1,1855 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Text;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.DotNet.Execute;
+
+using System.Reflection;
+
+namespace Microsoft.Fx.CommandLine
```

Comment:
I started changing the file with commit `Improve help usage`. If you go to that commit you could differ the files.It has most of the changes.
The next commits are small fixes or typos.